test/fixtures.rb

```diff
@@ -46,3 +46,33 @@ class ArrayMailer
 
   to ["noreply1@example.com", "noreply2@example.com"]
 end
+
+class DeliveryMethodMailer
+  include Lotus::Mailer
+  MyCustomDeliveryMethod = :smtp
+
+  Lotus::Mailer.configure do
+    delivery_method MyCustomDeliveryMethod, foo: 'bar'
+  end
+
+  mail = Mail.new
+  mail.delivery_method *Lotus::Mailer.configuration.delivery_method
+end
+
+class WelcomeMailer
+  include Lotus::Mailer
+
+  def greeting
+    "Ahoy"
+  end
+end
+
+class User
+  def initialize(locals)
+    @name = locals[:name]
+    @username = locals[:username]
+  end
+
+  attr_reader :name
```

Comment:
this and the next line can be combined into 1 line `attr_reader :name, :username`